cmd/copyUtil.go

```diff
@@ -85,6 +85,11 @@ func stripTrailingWildcardOnRemoteSource(source string, location common.Location
 		return
 	}
 
+	if strings.HasSuffix(resourceURL.RawPath, "//*") {
```

Comment:
Good catch on the issue itself, but I think we can actually solve this fundamentally rather than just flagging it.

I have a solution in mind (backed by my own testing), but I'd like to see if you can take a crack at it first. Lemme know if you need any help.